### PR TITLE
New Tutorial: Securely Access Resources with IAM Service Roles

### PR DESCRIPTION
**Pull request by spara**
# Content Pull Request Checklist

Please check to make sure your content submission meets the following requirements by adding an `x` between the brackets:

- [x ] The title of this pull request reflects the title of my content.
- [x ] I have reviewed the [Content Review Checklist](/content/blob/main/CONTENT_REVIEW_CHECKLIST.md).
- [ x] This pull request contains one piece of content (post or page).

**Important: if you are submitting a PR to update existing content, please ensure you [sync'ed your fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork) first before you created your branch.**

Is this pull request linked to a Content Proposal or Idea Suggestion Issue? If so, please link to it below using the number or full link.

```text
Resolves #5

or

Content for https://github.com/build-on-aws/content/issues/5
```

**By submitting this pull request, I confirm I own this content, have permission to use each image included in this content, and that it will be released under the [CC BY-SA 4.0 SA License](/LICENSE).**

### CODE REVIEW

**Comment by cobusbernard** [comment]
@spara, can you ping me please when the CFN template link has been updated? Also note that I did a rebase from `main` to get the latest `series.md` to link this piece to it when you make any changes on your checked out branch.

File: tutorials/practical-cloud-guide/securely-access-resources-with-iam-service-roles/index.md [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+
+## What’s Next
+
+Our next article will cover storage and the different forms of cloud storage. We will examine the primary use cases for different types of storage and how to set them up. In the mean time, you can read about the different types of cloud storage and when to [use them]
+(https://community.aws/concepts/how-to-store-data-in-the-cloud?sc_channel=el&sc_campaign=tutorial&sc_content=itpros&sc_geo=mult&sc_country=global&sc_outcome=acq&sc_publisher=amazon_media&sc_category=other&sc_medium=body).
+
+
+
+
+Also end with this line to ask for feedback:
```
[the remark this answers is not in the document]
Comment (spara) [reply tutorials/practical-cloud-guide/securely-access-resources-with-iam-service-roles/index.md]: fixed

**Comment by cobusbernard** [comment]
@cherbk ready for editorial review.

**Review by cobusbernard — APPROVED** [review]

**Comment by scribwell** [comment]
@DalM2023 This tutorial is ready for editorial review.

**Review by DalM2023 — COMMENTED** [review]
1st round edit done. Ready for review.

**Comment by scribwell** [comment]
@spara We've finished the first pass of our editorial review. Please either accept our editor's changes or provide a reason for rejecting them and make any additional necessary edits by October 2. Thank you!

File: tutorials/practical-cloud-guide/securely-access-resources-with-iam-service-roles/index.md [hunk 2]
```diff
@@ -0,0 +1,285 @@
...
+![Select the role and delete](./images/clean-up-2.png)
+
+## What We Did
+
+At the start of this tutorial we set up out environment, however this is not the focus of the tutorial. Cloudformation is a powerful tool  but we will cover it in following articles. The focus of this tutorial how to securely access resources with roles and temporary credentials. We created a role for a service and applied a policy that allowed it to access another resource. In this case, we attached a role to an EC2 instance that let it access an S3 bucket. We also saw that we can edit a role and add an action in this simple example.We could also add other statements, such as providing access to a relational database. The takeaway is to use roles to enable actions without exposing user credentials by placing them in service such as EC2.
+
+## What’s Next
+
+Our next article will cover storage and the different forms of cloud storage. We will examine the primary use cases for different types of storage and how to set them up. In the mean time, you can read about the different types of cloud storage and when to [use them](/concepts/how-to-store-data-in-the-cloud).
+
```
[the remark this answers is not in the document]
Comment (spara) [reply tutorials/practical-cloud-guide/securely-access-resources-with-iam-service-roles/index.md:286]: links added

File: tutorials/practical-cloud-guide/securely-access-resources-with-iam-service-roles/index.md [hunk 3]
```diff
@@ -0,0 +1,285 @@
...
+In the IAM console, select Roles on the side menu, select the role we created, and choose **Delete**.
+
+![Select the role and delete](./images/clean-up-2.png)
+
+## What We Did
+
+At the start of this tutorial we set up out environment, however this is not the focus of the tutorial. Cloudformation is a powerful tool  but we will cover it in following articles. The focus of this tutorial how to securely access resources with roles and temporary credentials. We created a role for a service and applied a policy that allowed it to access another resource. In this case, we attached a role to an EC2 instance that let it access an S3 bucket. We also saw that we can edit a role and add an action in this simple example.We could also add other statements, such as providing access to a relational database. The takeaway is to use roles to enable actions without exposing user credentials by placing them in service such as EC2.
+
+## What’s Next
```
[the remark this answers is not in the document]
Comment (spara) [reply tutorials/practical-cloud-guide/securely-access-resources-with-iam-service-roles/index.md:283]: question mark added

**Comment by scribwell** [comment]
@damonk10 Would you please review @spara's edits as part of a combined second/final review so we can expedite publishing?

**Review by damonk10 — APPROVED** [review]

File: tutorials/practical-cloud-guide/securely-access-resources-with-iam-service-roles/index.md [hunk 4]
```diff
@@ -0,0 +1,287 @@
...
+tags:
+    - service-roles
+    - it-pros
+    - tutorials
+    - aws
+    - iam
+authorGithubAlias: spara
+authorName: Sophia Parafina
+date: 2023-10-06
+showInHomeFeed: true
```
Comment (cobusbernard) [inline tutorials/practical-cloud-guide/securely-access-resources-with-iam-service-roles/index.md:13]: @spara : You can remove this line in future content pieces, we updated the configs to auto-include all content from our content folders in git.

Comment (cobusbernard) [inline tutorials/practical-cloud-guide/securely-access-resources-with-iam-service-roles/index.md:13]:
```suggestion
```

File: tutorials/practical-cloud-guide/securely-access-resources-with-iam-service-roles/index.md [hunk 5]
```diff
@@ -0,0 +1,287 @@
...
+---
+
```
Comment (cobusbernard) [inline tutorials/practical-cloud-guide/securely-access-resources-with-iam-service-roles/index.md:15]:
```suggestion
|ToC|
|---|
```

**Review by cobusbernard — APPROVED** [review]

**Pull request merged**